csfieldguide/static/interactives/password-guesser/passwords.txt

```diff
@@ -0,0 +1,13 @@
+These are the passwords encrypted in advance for the interactive.
```

Comment:
Could be nice to have Alice's, Bob's, Casey's and Dave's passwords in this file too even though they are in `passwords.js`